src/components/operations/operation-details/ko/runtime/operation-console.ts

```diff
@@ -73,7 +74,8 @@ export class OperationConsole {
         private readonly tenantService: TenantService,
         private readonly httpClient: HttpClient,
         private readonly routeHelper: RouteHelper,
-        private readonly settingsProvider: ISettingsProvider
+        private readonly settingsProvider: ISettingsProvider,
+        private readonly logger: ConsoleLogger
```

Comment:
This needs to be interface, not specific logger: `private readonly logger: Logger`. This way by changing the logger registration, we can use different loggers depending on environment.

Reply:
Got it, changed to interface